packages/shared/src/components/auth/EmailCodeVerification.tsx

```diff
@@ -108,7 +108,7 @@ function EmailCodeVerification({
             onClick={onSendCode}
             disabled={resendTimer > 0}
           >
-            {resendTimer === 0 ? 'Resend' : `Resend code ${resendTimer}s`}
+            {resendTimer === 0 ? 'Resend' : `Resend code: ${resendTimer}s`}
```

Comment:
How do we feel about making the `${resendTimer}s` use a monospace font? That way the button doesn't change size whenever 

https://github.com/user-attachments/assets/48391811-f89e-4a5f-8a7c-8d895e7e4144

Reply:
We can log it as a design ticket for backlog not urgent enough for now I'd say.

Reply:
https://dailydotdev.slack.com/archives/C07EA1E1XME/p1724667240167429

Reply:
done